README.md

```diff
@@ -1 +1,52 @@
-# the-concert-project
\ No newline at end of file
+# the-concert-project
+
+## Description
+
+- This project is about building a site for an event set up and mobile version first part is creating an entry page to the site of the event
+- The second part is creating speakers exhibition pages and a list of their names titles and description of the program.
+- The third part is the about part which gives an enlightment on the project.
+- The forth part is applying media queries to the site to make the mobile version and desktop version available on different devices and to enhance th UX of the site.
+- The burger menu part aims to create and open a nav menu when clicked on hamburger icon.
+- the pop-up windows part aims to show a pop up window once the user clicks on the burger menu icon at project section.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- node-modules
+- lighthouse
+- hintweb
+- stylint
+
+## links
+
+- https://basem909.github.io/microverse-portofolio-project-/
+  this is the link to the page on internet.
+
+## Authors
+
+👤 **Bassem Shams**
+
+- GitHub: [@githubhandle](https://github.com/basem909)
+
+## credits to
+
+Cindy Shin: The original designer at Behance
+
+the link tothe designer's profile:
+https://www.behance.net/adagio07
+
+## loom video link
+
+https://www.loom.com/share/a77b61cf7f8041f1b0b0cb70e8defcff
```

Comment:
- [ ]  Kindly make sure to highlight a piece of the code you find interesting or proud of making in the video presentation.